### DISCUSSION TITLE
Miyoogamelist.xml

### DISCUSSION THREAD

**Discussion by ac3nl** [OnionUI/Onion#532] [General]
Does anyone have made such a file
For a complete mame2003plus set?

Wish to share?

**Comment by Olywa** [comment]
I strongly recommend making your own, using Skraper and the guide in the WIki (FAQs) as the results will represent YOUR full set. If you have already scraped your games for images with Skraper then it's likely you already have a 'gamelist.xml' which is very easy to convert (again using the guide).

Due to several factors my 'full set' may be different to yours (from a different source, been altered at all, whether it's a split/merged/non-merged set...).   

If none of that concerns you, you can try this one (It is, I believe a full 'non-merged' set, covering 4924 games for Mame2003+):
[miyoogamelist.zip](https://github.com/jimgraygit/Onion/files/8773126/miyoogamelist.zip)

**Reply by ac3nl** [reply]
Thank you, this is perfect!

**Reply by ac3nl** [reply]
> I strongly recommend making your own, using Skraper and the guide in the WIki (FAQs) as the results will represent YOUR full set. If you have already scraped your games for images with Skraper then it's likely you already have a 'gamelist.xml' which is very easy to convert (again using the guide).
> 
> Due to several factors my 'full set' may be different to yours (from a different source, been altered at all, whether it's a split/merged/non-merged set...).
> 
> If none of that concerns you, you can try this one (It is, I believe a full 'non-merged' set, covering 4924 games for Mame2003+): [miyoogamelist.zip](https://github.com/jimgraygit/Onion/files/8773126/miyoogamelist.zip)

I did skrape my own, and did the strip with xml.exe. 
it doesn”t seem to work, i still have a lot of filenames, instead of game names.
Does ist matter that i have too many games (full set) and all the covers?

how big may an xml be, before miyoo doesn’t display it anymore?
how can you see that Miyoo loadedd the gamelist ?

**Reply by Olywa** [reply]
There's no size limit that I am aware of. If you are seeing filenames the list has not been loaded. 
Which system are you having issues with (still mame?). 
By default Onion is only configured for it to work for Arcade based systems but it's a simple tweak of a config file to have it working on other systems/consoles.

Does the XML (after you stripped it) still have Rom name, Game name & Image path? It must have all three to work. 
Did you see the guide here: https://github.com/jimgraygit/Onion/wiki/3.-FAQ-(Frequently-asked-questions)
